number/src/serialize.rs

```diff
@@ -0,0 +1,73 @@
+use std::io::{Read, Write};
+
+use crate::{BigInt, DegreeType, FieldElement};
+
+pub fn write_polys_file<T: FieldElement>(
+    file: &mut impl Write,
+    degree: DegreeType,
+    polys: &Vec<(&str, Vec<T>)>,
+) {
+    let width = f32::ceil(T::modulus().to_arbitrary_integer().bits() as f32 / 64f32) as usize * 8;
```

Comment:
Why is this a multiple of 64 bits?

Reply:
So this is a bit hacky but basically field elements are stored as arrays of u64, so this line gets the number of bytes to read for one field element.